sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -521,50 +646,82 @@ def _to_internal_entity(self):
         return self._internal_sd
 
 
-class SubscriptionRuntimeInfo(object):
-    """Description of a Service Bus queue resource.
+class SubscriptionRuntimeProperties(object):
+    """Runtime properties of a Service Bus topic subscription resource.
 
     :ivar str name:
-    :ivar created_at: The exact time the queue was created.
+    :ivar created_at: The exact time the subscription was created.
     :type created_at: ~datetime.datetime
     :ivar updated_at: The exact time a message was updated in the queue.
     :type updated_at: ~datetime.datetime
     :ivar accessed_at: Last time a message was sent, or the last time there was a receive request
      to this queue.
     :type accessed_at: ~datetime.datetime
-    :ivar message_count: The number of messages in the subscription.
-    :type message_count: int
-    :ivar message_count_details: Details about the message counts in queue.
-    :type message_count_details: ~azure.servicebus.management._generated.models.MessageCountDetails
+    :ivar total_message_count: The number of messages in the subscription.
+    :type total_message_count: int
+    :ivar active_message_count: Number of active messages in the subscription.
+    :type active_message_count: int
+    :ivar dead_letter_message_count: Number of messages that are dead lettered.
+    :type dead_letter_message_count: int
+    :ivar transfer_dead_letter_message_count: Number of messages transferred into dead letters.
+    :type transfer_dead_letter_message_count: int
+    :ivar transfer_message_count: Number of messages transferred to another queue, topic, or
+     subscription.
+    :type transfer_message_count: int
```

Comment:
property vs instance variable

Reply:
Removed ivar in dostrings and add docstrings to each property separately